### PR TITLE
Update ANZT tournaments

### PR DESCRIPTION
**Pull request by JWWolstenholme**
- Added ANZT5W page
- Updated ANZT5S page

### CODE REVIEW

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+Aus/NZ osu!standard Tournament 5 - Winter
+===
+
+*See also: [Aus/NZ osu!standard Tournament 5 - Summer](/wiki/Tournaments/ANZT/2018_Summer/en.md)*
+
+![Osu! AU/NZ Tournament 5 Winter](logo.jpg)
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]: Make a new folder named `img`
And place the image inside it

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+**Aus/NZ osu!standard Tournament 5 - Winter** (**ANZT 5W**) is a double elimination 1v1 osu!standard tournament hosted by ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475) and ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135). It is the 6th installment of Aus/NZ tourneys.
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
The **Aus/NZ osu!standard Tournament 5 - Winter** (***ANZT 5W***) was a double elimination one-on-one osu!standard tournament hosted by ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475) and ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135). It was the 6th installment of Aus/NZ tourneys.
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 28 Jul 2018 - 12 Aug 2018 |
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
| Registration Phase | 2018-07-28/2018-08-12 |
```
Use `YYYY-MM-DD` date format, apply on other instances

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| Streamers | ![][flag_AU] [Astellis](https://osu.ppy.sh/users/8369780), ![][flag_AU] [Bauxe](https://osu.ppy.sh/users/1881685), ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859) |
+| Referees | ![][flag_AU] [loler123903](https://osu.ppy.sh/users/8034438), ![][flag_AU] [Astellis](https://osu.ppy.sh/users/8369780), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425) |
+| Mappool Selectors | ![][flag_AU] [Elsa Valentine](https://osu.ppy.sh/users/4525153), ![][flag_US] [LowAccuracySS](https://osu.ppy.sh/users/7306763), ![][flag_AU] [Quite Cynical](https://osu.ppy.sh/users/3361154), ![][flag_AT] [Shiinoha Kotone](https://osu.ppy.sh/users/5712451), ![][flag_US] [noctisv_v](https://osu.ppy.sh/users/6775083), ![][flag_CA] [Legless](https://osu.ppy.sh/users/3224243) |
+| Graphics Design | ![][flag_US] [Monko2k](https://osu.ppy.sh/users/4852013) |
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/784018)
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/784018)
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| :-: | --- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![][flag_AU] [Dumii](https://osu.ppy.sh/users/3068044) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_AU] [-Machine-](https://osu.ppy.sh/users/5459981) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_AU] [Blobby3000](https://osu.ppy.sh/users/6916774) |
+
+All individual match results can be found [Here](https://docs.google.com/spreadsheets/d/1ylrPqTzCRTyQQG_RGBqZJZrUz_QFgSs_v26SFwhqLAs)
+
+---
+
+## Ruleset
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]: Add 1 blank space after every heading. Apply for other instances

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 6]
```diff
@@ -22,29 +24,29 @@ Aus/NZ osu!standard Tournament 5 - Summer
...
-| Referees | ![][flag_US] [noctisv_v](https://osu.ppy.sh/u/6775083), ![][flag_NZ] [Diony](https://osu.ppy.sh/u/4294475), ![][flag_AU] [RoseAntebellum](https://osu.ppy.sh/u/3944288), ![][flag_AU] [Illinon](https://osu.ppy.sh/u/8306102), ![][flag_AU] [Vex](https://osu.ppy.sh/u/1151852), ![][flag_AU] [Poi Yasu](https://osu.ppy.sh/u/6733334), ![][flag_AU] [Quite Cynical](https://osu.ppy.sh/u/3361154), ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/u/3772135), ![][flag_AU] [Melt3dCheeze](https://osu.ppy.sh/u/634837), ![][flag_NZ] [-Akito](https://osu.ppy.sh/u/5716327), ![][flag_AU] [BranOtaku](https://osu.ppy.sh/u/6861227), ![][flag_US] [trey-](https://osu.ppy.sh/u/8784747) |
-| Map Selectors | ![][flag_AU] [Dumii](https://osu.ppy.sh/u/3068044), ![][flag_US] [Axarious](https://osu.ppy.sh/u/2614511), ![][flag_US] [noctisv_v](https://osu.ppy.sh/u/6775083), ![][flag_AU] [Quite Cynical](https://osu.ppy.sh/u/3361154) |
-| Graphics Design | ![][flag_NZ] [-Akito](https://osu.ppy.sh/u/5716327) |
+| --- | --- |
```
[the remark this answers is not in the document]
Comment (Joehuu) [reply wiki/Tournaments/ANZT/2018_Summer/en.md]: Use left align with the colons (not center):
`| :-- | :-- |`

**Comment by JWWolstenholme** [comment]
Appreciate the help @fajar13k!

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 7]
```diff
@@ -1,50 +1,52 @@
 Aus/NZ osu!standard Tournament 5 - Summer
-=============================
+===
 
-![Osu! AU/NZ Tournament 5 Summer](logo.jpg)
+*See also: [Aus/NZ osu!standard Tournament 5 - Winter](/wiki/Tournaments/ANZT/2018_Winter/en.md)*
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
*See also: [Aus/NZ osu!standard Tournament 5 - Winter](/wiki/Tournaments/ANZT/2018_Winter)*
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 8]
```diff
@@ -1,50 +1,52 @@
 Aus/NZ osu!standard Tournament 5 - Summer
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
# Aus/NZ osu!standard Tournament 5 - Summer
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 9]
```diff
@@ -1,50 +1,52 @@
...
-=============================
+===
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 10]
```diff
@@ -1,50 +1,52 @@
...
-| :---: | :--- |
-| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months osu!supporter tag, profile badge |
-| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months osu!supporter tag |
-| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month osu!supporter tag |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
 
 ## Organization
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
## Organisation
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 11]
```diff
@@ -1,50 +1,52 @@
...
 
 | Position | Staff Members |
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
| Position | Member(s) |
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 12]
```diff
@@ -1,50 +1,52 @@
...
 ## Links
 
-- [Forum thread](https://osu.ppy.sh/forum/t/679243)
+- [Discussion thread](/community/forums/topics/679243)
 - [Livestream](https://www.twitch.tv/osuanzt)
 
------------------------
+---
 
 ## Mappool
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
## Mappools
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 13]
```diff
@@ -0,0 +1,111 @@
+Aus/NZ osu!standard Tournament 5 - Winter
+===
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 14]
```diff
@@ -0,0 +1,111 @@
+Aus/NZ osu!standard Tournament 5 - Winter
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
# Aus/NZ osu!standard Tournament 5 - Winter
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+===
+
+*See also: [Aus/NZ osu!standard Tournament 5 - Summer](/wiki/Tournaments/ANZT/2018_Summer/en.md)*
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
*See also: [Aus/NZ osu!standard Tournament 5 - Summer](/wiki/Tournaments/ANZT/2018_Summer)*
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
+
+## Organization
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
## Organisation
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 17]
```diff
@@ -0,0 +1,111 @@
...
+
+| Position | Staff Members |
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
| Position | Member(s) |
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 18]
```diff
@@ -0,0 +1,111 @@
...
+* Participants must have an Australian or New Zealand flag on their osu profile.
+* Matches will be **Head to Head** with **Score V2** as the scoring system.
+* Any harassment or abuse of a staff member or other players will result in an expulsion from the tournament.
+* If a problem occurs where rules are not clear, use common sense. Further questions can be directed to admins. Administration has final say regarding these situations.
+* In order for your registration to count, you must **fill out the form, join the discord, and change your name to your osu! username**.
+* Staff members are not allowed to sign up. The only exceptions are streamers & commentators.
+* Reschedules must be agreed on by **both players and proof must be provided**.
+* All matches will be scheduled for Fridays, Saturdays & Sundays.
+
+### Match Rules:
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
### Match Rules
```

File: wiki/Tournaments/ANZT/2018_Winter/en.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+* Disconnects may only occur once per match per player. It will count as a forfeit after the first time.
+* Highest !roll has first warmup, ban and pick.
+* Each player is allowed one ban.
+* Each player is allowed one warmup. Warmups must not be longer than 5 minutes, must not be in any ANZT5W mappool and must be submitted.
+* One player may not pick from the same mod pool twice in a row (HD/HR/DT/FreeMod). This does not apply to NoMod.
+* In a FreeMod pick, each player **must select a mod(s)** (HD/HR/HDHR). **NoMod is not allowed**.
+* Failed scores will only count if the player revives before the end. If a player fails and their opponent does not, the point goes to the latter. If both players fail, the point goes to the higher failed score.
+* If the match ends in a tie, players will play the tiebreaker. The tiebreaker will be played under FreeMod conditions, allowing NoMod.
+
+### Winning Conditions:
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Winter/en.md]:
```suggestion
### Winning Conditions
```

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 20]
```diff
@@ -323,6 +329,7 @@ Results of the main bracket can be found [Here](http://anzttournaments.challonge
 * **There will be no warmups and one ban per player**.
 
 ### Winning conditions
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]:
```suggestion
### Winning Conditions
```

**Comment by JWWolstenholme** [comment]
Appreciate the suggestions @Joehuu!

**Comment by JWWolstenholme** [comment]
Thanks again.

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 21]
```diff
@@ -1,52 +1,53 @@
...
 
-- [Forum thread](https://osu.ppy.sh/forum/t/679243)
+- [Discussion thread](/community/forums/topics/679243)
 - [Livestream](https://www.twitch.tv/osuanzt)
 
------------------------
+---
 
-## Mappool
+## Mappools
```
Comment (WalterToro) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]: Mappools should go from newest to oldest (invert the list)

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 22]
```diff
@@ -1,52 +1,53 @@
...
 
 ### Qualifiers
```
Comment (WalterToro) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]: The mappool pack messages should follow the next format: Download the mappack here! (XX MB).

**Comment by JWWolstenholme** [comment]
@WalterToro Thanks. Added those changes

**Comment by JWWolstenholme** [comment]
@WalterToro, I think you need to approve the changes I made or something for the PR to show up as mergeable, thanks.

File: wiki/Tournaments/ANZT/2018_Summer/en.md [hunk 23]
```diff
@@ -1,36 +1,37 @@
-Aus/NZ osu!standard Tournament 5 - Summer
-=============================
+# Aus/NZ osu!standard Tournament 5 - Summer
 
-![Osu! AU/NZ Tournament 5 Summer](logo.jpg)
+*See also: [Aus/NZ osu!standard Tournament 5 - Winter](/wiki/Tournaments/ANZT/2018_Winter)*
```
Comment (cl8n) [inline wiki/Tournaments/ANZT/2018_Summer/en.md]: are these two Summer/Winter ANZTs any more related to each other than the rest of the ANZT series (other than the number)? if not, I don't think you should include this "See also" note

**Comment by JWWolstenholme** [comment]
Tbh I'm not sure how to update this PR since I've deleted the fork. I've managed to clone this PR but can't push anything back without maintainer. Any advice?

The conflict is simple and both sides of the conflict need to be saved if that helps anyone.

**Comment by JWWolstenholme** [comment]
Closed in favor of #2367.

**Pull request closed without merging**